docs/notes/hybrid_and_offline_rl_training.rst

```diff
@@ -9,13 +9,14 @@ capability of ALF.
 --------------------------------------------------------
 Here we only focus on the RL training functionality of ALF. For this, we
 have several different training modes supported in ALF:
+
```

Comment:
Previous missing a new line in L12 and also in L77, making the format of the rendered doc broken.